### ISSUE TITLE
Escaping Locked lockers ,abillity to break out even when moving.

### ISSUE THREAD

**Issue by Razorblaza** [HippieStation/HippieStationdeprecated2020#8702]
[Implementation]: Currently ingame it takes 2 minutes to break out of a locker pretty short right, well the problem is that if the locker is moved the whole process restarts what im suggesting is that you should be able to break out of lockers while moving if your not wearing handcuffs or something like that

[Why]:Because currently someone could just put you in a locker then lock it and then just drag you away if they are constantly moving you have no way to escape, also it doesn't make that much sense that you cannot bang against a locker door while it is moving, i think this would just stop people from getting a locker speed locking it then just carrying off with someone.

[Resources]: None needed i believe

# Be sure to ask some maintainer to label this issue as a feature. Either ping them on our discord or PM them on our forum. Thank you for sharing your ideas!
@HippieStation/maintainers

**Comment by JohnGinnane** [comment]
\>breaking out of lockers while it's moving
Jesus yeah I want this, it doesn't make sense to be unable to do this

**Comment by JohnGinnane** [comment]
@Carbonhell your ping didn't work :^)

**Comment by Carbonhell** [comment]
i tried

**Comment by ghost** [comment]
yes pls +1

**Comment by ghost** [comment]
Add banging lockers to alert nearby people +1

-1 for this idea as it is now

**Comment by JohnGinnane** [comment]
@fluxcapacitor1337 I'm pretty sure it does alert people but other that there is no escaping it, not even freedom implant or acid thing

**Comment by Carbonhell** [comment]
I agree that it makes no sense to not be able to free yourself, the delay is already long enough, no need to make it pointless. @t-f-w thoughts

**Comment by Eagle-Eyess** [comment]
My only problem on this is how tf do you get yourself locked inside a locker without cuffs on? Maybe someone stuns you and puts you in but they could probably cuff you in that same amount of time. -1

**Comment by Carbonhell** [comment]
@t-f-w T H O U G H T S ?

**Comment by Carbonhell** [comment]
mfw no thoughts from tfw

**Comment by Razorblaza** [comment]
just gonna bump this because you've all forgot about it, this feature could stop alot of griefers

**Comment by Jujumatic** [comment]
should be easy to implement, $1 bounty most likely

**Comment by AnthonyJNgo** [comment]
$2 bounty, showing our coders love.

**Comment by xTrainx** [comment]
Where's my money, bookie?
